java/src/jmri/Timebase.java

```diff
@@ -119,8 +120,11 @@
 
     // methods for start up with clock stopped option
     public void setStartStopped(boolean stopped);
-
     public boolean getStartStopped();
+    // methods for start up with clock running option. If neither stopped nor running is set, the
+    // clock run state will not change at startup.
+    public void setStartRunning(boolean running);
```

Comment:
`setStartStopped` and `setStartRunning` seem like unfortunate names to me.  (Though I understand that setStartStopped is historical). It's not clear from the name what they do.  Could we have `setInitiallyRunning` and `setInitiallyStopped`?

Or perhaps even `setInitialState(enum Running;Stopped;Unchanged)`?  That would also avoid the necessary interaction between setStartStopped and SetStartRunning  (you can' have both set), which is another form of confusing side-effect.

Reply:
Good idea. Done.